lib/routes/szftedu/gonggao.ts

```diff
@@ -0,0 +1,63 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const host = 'https://ylxx.szftedu.cn/xx_5828/xygg_5832/';
+
+
+export const route: Route = {
+    path: '/gonggao',
+    categories: ['other'],
+    example: '/szftedu/gonggao',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '园岭小学公告',
```

Comment:
No need to repeat namespace name in route name.

Reply:
suggestion committed.